node/node.go

```diff
@@ -70,6 +67,7 @@ var (
 //  is mediated by methods.
 
 type Node struct {
+	CTX                     context.Context
```

Comment:
`CTX`?

Reply:
needed because existing tests in a different package need direct variable access